decidim-comments/spec/features/comments_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe "Comments", type: :feature do
+  let!(:organization) { create(:organization) }
+  let!(:user) { create(:user, :confirmed, organization: organization) }
+  let!(:participatory_process) { create(:participatory_process, organization: organization) }
+  let!(:comments) {
+    3.times.map do
+      create(:comment, commentable: participatory_process)
+    end
+  }
+
+  before do
+    switch_to_host(organization.host)
+  end
+
+  it "user should see a list of comments" do
+    visit decidim.participatory_process_path(participatory_process)
```

Comment:
We should probably be testing this feature in a dummy resource that only gets loaded in `test` in order not to couple it anywhere.